packages/client/src/runtime/core/engines/library/WasmLibraryLoader.ts

```diff
@@ -21,12 +21,19 @@ export const wasmLibraryLoader: LibraryLoader = {
       )
     }
 
+    if (engineWasm === undefined) {
+      throw new PrismaClientInitializationError(
+        `WASM engine loading was not configured correctly. Please, re-run \`prisma generate\``,
```

Comment:
How could `prisma generate` fix the issue?

Reply:
Leftover from an earlier version of the code where `runtime` was not nested under wasm and it was possible to use older client with newer runtime accidentally. Now you are right, the message is misleading, I'll change it.